src/libs/wagmi/wagmi.constants.ts

```diff
@@ -0,0 +1,36 @@
+import { ChainIdMapTo, SelectableConnectionType } from 'libs/wagmi/wagmi.types';
+import config from 'config';
+import { tenderlyRpc } from 'utils/tenderly';
+
+const CHAIN_RPC_URL = tenderlyRpc || config.network.rpc.url;
+const CHAIN_RPC_HEADERS = tenderlyRpc ? {} : config.network.rpc.headers || {};
+const CHAIN_ID = config.network.chainId;
+
+if (typeof CHAIN_RPC_URL === 'undefined') {
+  throw new Error(`rpcUrl must be defined in config folder`);
+}
+
+export const IS_TENDERLY_FORK = !!tenderlyRpc;
+
+export enum EnumConnectionType {
+  METAMASK,
+  COINBASE_WALLET,
+  WALLET_CONNECT,
+  GNOSIS_SAFE,
+}
+
+export enum SupportedChainId {
+  MAINNET = CHAIN_ID,
+}
```

Comment:
Use a regular object instead 
```suggestion
export enum SupportedChainId {
  MAINNET = CHAIN_ID,
}
```
```suggestion
export const SupportedChainId = {
  mainnet: CHAIN_ID,
}
```